### DISCUSSION TITLE
Reconnecting to server

### DISCUSSION THREAD

**Discussion by deathbull** [jchristn/WatsonWebsocket#70] [Q&A]
How can i reconnect after disconnect from server. After disconnection, StartWithTimeoutAsync or StartWithTimeout doesn't work.

**Comment by jchristn** [comment]
Hi @deathbull in the Test.Server and Test.Client project you can give this a try.  I started the server, connected the client, ```Dispose()```d the client, then reconnected the client (which reinstantiates the client, i.e. ```client = new WatsonWsClient(...```.

![image](https://user-images.githubusercontent.com/1421081/133474665-f3bd89ff-3dc5-4609-b98e-50ab096a1ca1.png)

**Comment by deathbull** [comment]
Oh sorry my bad i saw that unity lost the object after loaded the scene 1 more time, thank you again.

**Reply by jchristn** [reply]
Of course!  Have a good one my friend
